### PR TITLE
Add food

### PR DESCRIPTION
**Pull request by gzzsound**
to be renamed...

Copied and pasted the code to have something on the screen that we should collect.
We should probably remove the duplication.

BTW what is the name of this thing we need to collect?

### CODE REVIEW

**Review by lazydevorg — APPROVED** [review]

File: src/sketch.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const Flatten = globalThis["@flatten-js/core"];
-//const {segment} = Flatten;
+const {Point} = Flatten;
```
Comment (unokitchen) [inline src/sketch.js:2]: Point will conflict with P5js one

**Comment by gzzsound** [comment]
@unokitchen @lazydevorg I updated the logic:
- now food gets created only when a shadow dies
- new food gets created only when you collected the one that already exists

**Comment by lazydevorg** [comment]
@gzzsound I can't see the packages.json in this branch. Can you commit it?

**Comment by gzzsound** [comment]
@lazydevorg don't have it on the branch :(
Probably it was not committed when the http-server was added. @unokitchen can you please check?

**Comment by unokitchen** [comment]
npm install, I guess


Il giorno mer 5 ago 2020 alle ore 16:23 Alessandro Caianiello <
notifications@github.com> ha scritto:

**Comment by unokitchen** [comment]
ah without the json it will not work
let me check

Il giorno mer 5 ago 2020 alle ore 18:44 Mariano Cigliano <
unokitch3n@gmail.com> ha scritto:

**Comment by unokitchen** [comment]
npm install should actually work

Il giorno mer 5 ago 2020 alle ore 18:45 Mariano Cigliano <
unokitch3n@gmail.com> ha scritto:

**Pull request merged**